architecture/evm/eth_getLogs.go

```diff
@@ -82,6 +87,39 @@ func upstreamPreForward_eth_getLogs(ctx context.Context, n common.Network, u com
 		jrq.RUnlock()
 		return false, nil, nil
 	}
+
+	blockHash, ok := filter["blockHash"].(string)
+	if ok && blockHash != "" {
+		// EIP-234 => if blockHash is present, fromBlock or toBlock is NOT allowed
+		if _, hasFB := filter["fromBlock"]; hasFB {
+			jrq.RUnlock()
+			return true, nil, common.NewErrEndpointServerSideException(
+				fmt.Errorf("EIP-234 violation: blockHash and fromBlock cannot both be present for eth_getLogs"),
```

Comment:
These are client-side exceptions

Reply:
had it first as a client-side exception :) will fix